api/graphql/schema.graphql

```diff
@@ -1654,16 +1657,22 @@ type Post {
   myEventResponse: String
   # The reactions of the logged-in user to this
   myReactions: [Reaction]
-  # Whether the logged-in user has upvoted the post or not
-  myVote: Boolean
   # Number of people that have one or more reactions to a post
   peopleReactedTotal: Int
   # Number of total reactions on the post
   postReactionsTotal: Int
   # Number of total "members" of the post. Used for Projects only right now
   postMembershipsTotal: Int
+  # A proposal's outcome: 'cancelled', 'quorum-not-met', 'in-progress', 'successful', 'tie'
+  proposalOutcome: String
+  # The status of a proposal post: 'discussion', 'voting', 'completed'
+  proposalStatus: String
   # Link to a project management tool or service. Only used for Projects right now
   projectManagementLink: String
+  # The type of proposal: 'single', 'multiple-unrestricted', 'majority', 'consensus'
```

Comment:
we havent discussed building in "types" of majority or consensus. And I think the mapping of number of votes to outcome (like majority) is not actually the same type of info as whether you get one or more votes. for now we are not building in voting details like majority of consensus, we are just letting the creator specify what the outcome will be based on the votes and then marking it as so. right?